src/Plugins/ComplexCore/docs/ExecuteProcessFilter.md

```diff
@@ -0,0 +1,47 @@
+# Execute Process
+
+
+## Group (Subgroup) ##
+
+Core (Misc)
+
+## Description ##
+
+This filter allows the user to execute any application, program, shell script or any other executable program on the computer system. Any output can be found in the user specified log file.
+
```

Comment:
I would like to see some examples of what would be entered into the parameter. Even just simple things but showing that you should use the full absolute path, arguments to the executable.

For example for simple directory listing:
```
/bin/ls
```
for something with arguments
```
/bin/ls -laF
```
for something with a space in the path or name we need double quotes

```
cp "/tmp/Name of File.txt" "/tmp/Better_Name_of_File.txt"
```